projects/plugins/jetpack/package.json

```diff
@@ -77,6 +80,7 @@
 		"@wordpress/hooks": "2.12.3",
 		"@wordpress/i18n": "3.9.0",
 		"@wordpress/icons": "2.7.0",
+		"@wordpress/interface": "3.1.2",
```

Comment:
What is this going to be used for?

Reply:
Customberg relies heavily on [`@wordpress/interface`](https://github.com/WordPress/gutenberg/blob/2c924df03a45943eb5e21113b7295436e2facf90/packages/interface/README.md), which serves as scaffolding for Gutenberg-styled pages. You can preview what this might look like [here](https://github.com/Automattic/jetpack/pull/20209/files#diff-aa19933d080dd4d135719019b5e1b9b1c2351913eed9bbcf22e64f18e22834dfR11) from the original PR this was spun out from.